cachi2/core/utils.py

```diff
@@ -80,11 +84,83 @@ def load_json_stream(s: str) -> Iterator:
         yield obj
 
 
+def _get_blocksize(fd: int) -> int:
```

Comment:
A super-minor nitpick: this is unlikely to change for a single file, so could be cached.

Reply:
True.